app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ActionMailer::Base
+  default from: 'info@stavcdflashcard.com'
+
+  def welcome_email(user)
+    @user = user
+    @url =  'http://stavcdflashcards.herokuapp.com/login'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.